doc_src/index.hdr.in

```diff
@@ -846,6 +846,8 @@ The user can change the settings of `fish` by changing the values of certain var
 
 - `history`, an array containing the last commands that were entered.
 
+- `FISH_HISTFILE`: Name of the history session. It defaults to `$XDG_DATA_HOME/fish/fish_history` (typically `~/.local/share/fish/fish_history`). This variable can be changed by the user. It does not have to be an environment variable. You can change it at will within an interactive fish session to change where subsequent commands are logged.
```

Comment:
Looks like you forgot too update this message. The default should be "fish" not the entire path.